perfmetrics/scripts/show_results.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) != 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 show_results.py <fio output json filepath>')
+
+  # Opening JSON file
+  file = open(argv[1])
+
+  # returns JSON object as
+  # a dictionary
+  data = json.load(file)
+
+  # Iterating through the json
+  # list
+  for d in data['jobs']:
+      if d["read"]["bw"] != 0 :
+        if d["job options"]["rw"] == "read":
+          print("Filesize: "+ d["job options"]["filesize"])
```

Comment:
We should print filesize in all cases - read, write, randread and randwrite.

Reply:
It will print in this format
Filesize - 1
Read
Write
RandRead
RandWrite
Filesize - 2
Read
Write
RandRead
RandWrite

should I print filesize again and again in between ?

Reply:
I think how you are printing is fine.